HelpHunterBE/Controllers/RatingController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using HelpHunterBE.Dto;
+using HelpHunterBE.Logic;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HelpHunterBE.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    [Authorize]
+    public class RatingController : ControllerBase
+    {
+        private IRatingLogic _logic;
+
+        public RatingController(IRatingLogic logic)
+        {
+            this._logic = logic;
+        }
+
+        [AllowAnonymous]
```

Comment:
czy na pewno chcemy żeby cokolwiek oprócz logowania i rejestracji było dostępne bez autentykacji?

Reply:
Myślę że lista opinii o specjaliście może być publiczna, ale jeśli twierdzisz inaczej to mogę to zmienić

Reply:
IMO zróbmy za ścianą JWT żeby ograniczyć możliwość uderzania botów w endpointy - taka moja opinia, że ile się da zamknąć za JWT to tyle lepiej z racji, że nie ma w naszej aplikacji żadnych innych security ficzerów.

Reply:
W sumie w naszej aplikacji brakuje wielu security ficzerów - choćby to, że każdy zalogowany user może modyfikować dane innych userów xd ale ciiii, ważne żeby działało na razie